sites/website/src/docs/design-systems/high-contrast.md

```diff
@@ -0,0 +1,194 @@
+        background: ${accentFillRestBehavior.var};
+    }
+`
+```
+
+When high contrast is enabled, the system will try to apply the correct color. In the case of this accent button, the system is missing a few things. We do not have a background, rest and hover state is the same, focus is not following the button focus design, and the disabled state is too dim.
+
+![Accent button no forced colors](https://res.cloudinary.com/dm4izfqmy/image/upload/v1607550781/highContrast_examples/hc-document/acccent-no-forced-colors_h0peqd.png)
+
+To fix this, we will add `forcedColorsStylesheetBehavior` to `withBehaviors()`, using similar selectors, and adding the system color keyword.
+
+```css
+export const AccentButtonStyles = css`
+    /* ... */
+`.withBehaviors(
+    forcedColorsStylesheetBehavior(
+        css`
+            :host([appearance="accent"]) .control {
+                forced-color-adjust: none;
+                background: ${SystemColors.Highlight};
+                color: ${SystemColors.HighlightText};
+            }
+            :host([appearance="accent"]) .control:hover,
+            :host([appearance="accent"]:active) .control:active {
+                background: ${SystemColors.HighlightText};
+                border-color: ${SystemColors.Highlight};
+                color: ${SystemColors.Highlight};
+            }
+            :host([appearance="accent"]) .control:${focusVisible} {
+                border-color: ${SystemColors.ButtonText};
+                box-shadow: 0 0 0 2px ${SystemColors.HighlightText} inset;
+            }
+            :host([appearance="accent"][disabled]),
+            :host([appearance="accent"][disabled]) .control,
+            :host([appearance="accent"][disabled]) .control:hover {
+                background: ${SystemColors.ButtonFace};
+                border-color: ${SystemColors.GrayText};
+                color: ${SystemColors.GrayText};
+                opacity: 1;
+            }
+        `
+    )
+);
+```
+
+After adding `forced-colors` and applying the keywords, the accent button now has a background to the rest state, and we are using the `Highlight` color. On the hover and active state, the look is reversed from the rest state. Focus gets a double border treatment and disabled has opacity set to 1 and uses the disabled color, `GrayText`, on the border and content.
+
+![Accent button forced colors](https://res.cloudinary.com/dm4izfqmy/image/upload/v1607550781/highContrast_examples/hc-document/acccent-with-forced-colors_v76kif.png)
+
+### Further resources
+
+**Color contrast comparison chart**
+
+To help determine whether a pair of high contrast colors will meet a color luminosity contrast ratio of at least 10:1
+This table uses the high contrast theme color resource names you see in Windows Ease of Access.
+
+How to read this table:
+- <mark>YES</mark> - indicates that it is safe to assume this pair of colors will meet high contrast requirements, even in custom themes.
```

Comment:
Is this chart something we made or is it sourced from another location?

Reply:
This was a chart Rory had put together to help with keyword combinations. I had used it in the other wiki doc and thought of adding it here too.